common/src/main/java/com/wynntils/features/user/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -123,7 +123,7 @@ private void renderTexturedCount(PoseStack poseStack, int emeralds) {
 
         for (int i = emeraldAmounts.length - 1; i >= 0; i--) {
             String emeraldAmount = emeraldAmounts[i];
```

Comment:
Why is this a string and not an integer?

Reply:
So is the check for "0" always valid then? Maybe it is a french zero? ;-)

Reply:
It should be, yes.

Reply:
I think what I am saying is that we should not have two different ways to represent "no value".